lib/benchpark/system.py

```diff
@@ -82,7 +90,7 @@ def initialize(self):
         self.sys_cores_per_node = None
         self.sys_gpus_per_node = None
         self.sys_mem_per_node = None
-        self.scheduler = None
+        self.scheduler = self.spec.variants["scheduler"].value
```

Comment:
Once the comma is added to the definition, this will need to access via index:  `self.spec.variants["scheduler"][0]`

(this is a departure from how variants are referenced in Spack, but we don't seem to use `.value` anywhere in Benchpark)